scriptP2.js

```diff
@@ -1,28 +1,30 @@
-
-
 const base_url= "http://image.tmdb.org/t/p/";
 const getLastestApiUrl = 'https://api.themoviedb.org/3/movie/top_rated?api_key=60586031e0e21153f7f67ca901fabc85&language=en-US&page=1'
 const getPopularApiUrl = 'https://api.themoviedb.org/3/movie/popular?api_key=60586031e0e21153f7f67ca901fabc85&language=en-US&page=1';
 const api_key = '?api_key=60586031e0e21153f7f67ca901fabc85';
 
 
-function PopularMoviesGetter(){
-async function getPopularMovies (){   
-    let response = await fetch(getPopularApiUrl);
+fetchMovies(getLastestApiUrl).then(jsonProcessing);
```

Comment:
So we wanna fetch lates movies first time?

Reply:
yes, we cant to show it in frist instance, couse it will be the main page